packages/protocol/test/common/onboarding.ts

```diff
@@ -0,0 +1,424 @@
+  MockRandomInstance,
+  MockValidatorsContract,
+  ProxyCloneFactoryContract,
+  ProxyCloneFactoryInstance,
+  ProxyContract,
+  ProxyInstance,
+  RegistryInstance,
+  StableTokenInstance,
+} from 'types'
+
+const executeMetaTransaction = async (
+  signer: string,
+  mtw: MetaTransactionWalletInstance,
+  metaTx: MetaTransaction
+) => {
+  const { v, r, s } = await getSignatureForMetaTransaction(signer, mtw.address, metaTx)
+  return mtw.executeMetaTransaction(metaTx.destination, metaTx.value, metaTx.data, v, r, s)
+}
+
+const getExecuteMetaTransactionData = async (
+  signer: string,
+  mtw: MetaTransactionWalletInstance,
+  metaTx: MetaTransaction
+) => {
+  const { v, r, s } = await getSignatureForMetaTransaction(signer, mtw.address, metaTx)
+  // @ts-ignore
+  return mtw.contract.methods
+    .executeMetaTransaction(metaTx.destination, metaTx.value, metaTx.data, v, r, s)
+    .encodeABI()
+}
+
+const logCosts = false
+const logCost = (output: string) => {
+  if (logCosts) {
+    // tslint:disable-next-line:no-console
+    console.log(output)
+  }
+}
+
+const Accounts: AccountsContract = artifacts.require('Accounts')
+const Attestations: AttestationsTestContract = artifacts.require('AttestationsTest')
+const MTW: MetaTransactionWalletContract = artifacts.require('MetaTransactionWallet')
+const Random: MockRandomContract = artifacts.require('MockRandom')
+contract('Komenci Onboarding', (_accounts: string[]) => {
+  let accounts: AccountsInstance
+  let stableToken: StableTokenInstance
+  let mtw: MetaTransactionWalletInstance
+  let random: MockRandomInstance
+  let registry: RegistryInstance
+  let relayermtw: MetaTransactionWalletInstance
+  let transactions: any[]
+  const attestationExpiryBlocks = (60 * 60) / 5
+  const selectIssuersWaitBlocks = 4
+  const maxAttestations = 20
+  const user: string = _accounts[1]
+  // Random hex strings
+  const dek = '0x02f2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e01611111111'
+  const identifier = '0x02f2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e016111111'
+  const numAttestations = 3
+  // TODO(asa): Clean this up
+  // const perAttestationFee = web3.utils.toWei(numAttestations * 0.05, 'ether').toString()
+  const perAttestationFee = '50000000000000000'
+  const attestationFee = '150000000000000000'
+  before(async () => {
+    stableToken = await getDeployedProxiedContract('StableToken', artifacts)
+    // The komenci version is set as a test contract because we are not using it with a proxy.
+    relayermtw = await MTW.new(true)
+    await relayermtw.initialize(_accounts[0])
+
+    // Set up the required mocks that will allow verification to work with ganache.
+    registry = await getDeployedProxiedContract('Registry', artifacts)
+    random = await Random.new()
+    accounts = await Accounts.new()
+    await accounts.initialize(registry.address)
+    // Take ownership of the registry contract to point it to the mocks.
+    await assumeOwnership(['Registry'], _accounts[0])
```

Comment:
Isn't it already owned by the default deployer, which I think should be `_accounts[0]`?

Reply:
Oh interesting, I thought it was owned by governance. Is that no longer true for tests?

Reply:
Oh, if you're using the one deployed by migrations then you're right. I thought you were deploying a fresh Registry in a `beforeEach`. Note that if you do use the migrations one, this will break `yarn quicktest` for this suite.